src/gcs.cpp

```diff
@@ -118,30 +115,31 @@ void send_status_message() {
 		jsonVehicle["armed"] = vehicles[i]->armed;
 		jsonVehicle["tracking"] = vehicles[i]->tracking;
 
-		json jsonPosition = json::array();
-		jsonPosition.push_back(vehicles[i]->state.position.x());
-		jsonPosition.push_back(vehicles[i]->state.position.y());
-		jsonPosition.push_back(vehicles[i]->state.position.z());
-		jsonVehicle["position"] = jsonPosition;
+		jsonVehicle["position"] = {
```

Comment:
Nope, I'll put it back to the way it was (we aren't using the data at the moment anyway).